### PR TITLE
dev-games/aseprite: add 1.2.35

### PR DESCRIPTION
**Pull request by brausepulver**
Closes: https://bugs.gentoo.org/696186
Signed-off-by: Johannes Kindermann <binaryjockey@gmail.com>

### CODE REVIEW

File: dev-games/aseprite/aseprite-1.2.35.ebuild [hunk 1]
```diff
@@ -60,32 +66,42 @@ src_prepare() {
 		sed -i '/-DNDEBUG/d' CMakeLists.txt || die
 	fi
 	# Fix shebang in thumbnailer
-	sed -i -e 's:#!/usr/bin/sh:#!/bin/sh:' desktop/aseprite-thumbnailer || die
+	sed -i -e 's:#!/usr/bin/sh:#!/bin/sh:' src/desktop/linux/aseprite-thumbnailer || die
 }
 
 src_configure() {
-	use debug && append-cppflags -DDEBUGMODE -D_DEBUG
+	use debug && append-cppflags -DDEBUGMODE -D_DEBU
```
Comment (winterheart) [inline dev-games/aseprite/aseprite-1.2.35.ebuild:73]: Typo in -D_DEBU

**Review by winterheart — CHANGES_REQUESTED** [review]

**Comment by winterheart** [comment]
@brausepulver, we cannot drop 1.1.9 before 1.2.35 stabilization. Please change commit as only adding 1.2.35.

File: dev-games/aseprite/aseprite-1.2.35.ebuild [hunk 2]
```diff
@@ -1,56 +1,62 @@
...
+	x86? ( https://github.com/aseprite/skia/releases/download/m102-861e4743af/Skia-Linux-Release-x86.zip -> skia-x86.zip )
+	amd64? ( https://github.com/aseprite/skia/releases/download/m102-861e4743af/Skia-Linux-Release-x64.zip -> skia-x86_64.zip )"
 
 # See https://github.com/aseprite/aseprite#license
 # Some bundled third-party packages built-in:
 # gtest duktape modp_b64 simpleini
 LICENSE="Aseprite-EULA"
 SLOT="0"
-KEYWORDS="amd64 x86"
+KEYWORDS="~amd64 ~x86"
```
Comment (winterheart) [inline dev-games/aseprite/aseprite-1.2.35.ebuild:19]: Since there precompiled binaries, we need now `KEYWORDS="-* ~amd64 ~x86"` here

**Review by winterheart — CHANGES_REQUESTED** [review]

**Comment by brausepulver** [comment]
The latest version is reported to be 1.2.35.1 rather than 1.2.35 on https://packages.gentoo.org/packages/dev-games/aseprite. Where is this coming from?

**Comment by winterheart** [comment]
packages.gentoo.org takes info from Repology (https://repology.org/project/aseprite/versions), there wrongly assigned 1.2.35.1 from AUR repository.

**Comment by winterheart** [comment]
Hello.
I made another PR #26335 for aseprite 1.2.35, this can be closed.

**Pull request closed without merging**